docs/references/protocol/transactions/types/accountset.md

```diff
@@ -74,6 +74,7 @@ The available AccountSet flags are:
 |:----------------------------------|:--------------|:--------------|
 | `asfAccountTxnID`                 | 5             | Track the ID of this account's most recent transaction. Required for [`AccountTxnID`](../common-fields.md#accounttxnid) |
 | `asfAllowTrustLineClawback`       | 16            | Allow account to claw back tokens it has issued. _(Requires the Clawback amendment.)_ Can only be set if the account has an empty owner directory (no trust lines, offers, escrows, payment channels, checks, or signer lists). After you set this flag, it cannot be reverted. The account permanently gains the ability to claw back issued assets on trust lines. |
+| `asfAllowTrustLineLocking`        | 17            | Allow account to lock trust lines for **IOU** tokens it has issued. **IOU** Tokens: If the issuer's account does not have the `lsfAllowTrustLineLocking` flag set, the transaction fails with `tecNO_PERMISSION`. |
```

Comment:
"the `EscrowCreate` transaction fails with"  perhaps ? I guess this can be polished.